### ISSUE TITLE
use of const in strict mode doesn't work

### ISSUE THREAD

**Issue by shri** [bendrucker/generator-bd#2]
This breaks end users using node, as use of const in strict mode doesn't work. (Will work on iojs, due to different defaults).

``` sh
/usr/local/lib/node_modules/generator-bd/app/index.js:3
const yeoman = require('yeoman-generator')
^^^^^
SyntaxError: Use of const in strict mode.

```

**Comment by shri** [comment]
Fixed with #3

**Comment by bendrucker** [comment]
Published in 2.1.5
